documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -413,18 +413,18 @@
                "counter":0,
                "roughness":0.0
             },
-            "reads":{
+            "reads":{ // measures number of completed read requests
                "hz":0.0,
                "counter":0,
                "roughness":0.0
             },
-            "read_requests":{
+            "read_requests":{ // measures number of incoming read requests
                "hz":0.0,
                "counter":0,
                "roughness":0.0
             }
          },
-         "bytes":{ // of operations (independent of hz). Perfectly spaced operations will have a roughness of 1.0 . Randomly spaced (Poisson-distributed) operations will have a roughness of 2.0, with increased bunching resulting in increased values. Higher roughness can result in increased latency due to increased queuing.
+         "bytes":{ // measures number of logical bytes read/written (ignoring replication factor and overhead on disk). Perfectly spaced operations will have a roughness of 1.0 . Randomly spaced (Poisson-distributed) operations will have a roughness of 2.0, with increased bunching resulting in increased values. Higher roughness can result in increased latency due to increased queuing.
```

Comment:
I'm not familiar with this metric. 
Waiting for A.J.'s comment. :)

Reply:
I agree with this description; it counts the number of bytes being written on the proxies and being read from the storage servers based on the length of each key and value. Adding a suggested change to remove a space, though:

```suggestion
         "bytes":{ // measures number of logical bytes read/written (ignoring replication factor and overhead on disk). Perfectly spaced operations will have a roughness of 1.0. Randomly spaced (Poisson-distributed) operations will have a roughness of 2.0, with increased bunching resulting in increased values. Higher roughness can result in increased latency due to increased queuing.
```